<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovaComplexDialogClass</class>
 <widget class="QDialog" name="NovaComplexDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NovaComplexDialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>401</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>7</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>8</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>8</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>381</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Vendor</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the vendor who's MAC range you would like to use</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>400</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>To get started, enter the first few letters of the desired vendor into the box below and press search.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
